docs/source/DeveloperGuide/DevelopmentEnvironment/Eclipse/index.rst

```diff
@@ -3,7 +3,12 @@
 Configuring *Eclipse*
 =======================
 
-A possible way to develop in DIRAC is using `Eclipse <http://www.eclipse.org/>`_. We know `Emacs <www.gnu.org/s/emacs/>`_ is a great tool, and someone can't just leave without it. And that also `vim <http://www.vim.org/>`_ is great. Other excellent options include `pycharm <https://www.jetbrains.com/pycharm/>`_, `sublime <https://www.sublimetext.com/>`_, and `atom <https://atom.io/>`_.
+A possible way to develop in DIRAC is using `Eclipse <http://www.eclipse.org/>`_.
+We know `Emacs <www.gnu.org/s/emacs/>`_ is a great tool,
+and someone can't just leave without it.
+And that also `vim <http://www.vim.org/>`_ is great.
+Other excellent options include `pycharm <https://www.jetbrains.com/pycharm/>`_, `sublime <https://www.sublimetext.com/>`_,
+and `atom <https://atom.io/>`_.
```

Comment:
Can you please add that for the other editors, the formatting option should just be pep8 with 2 spaces ?